libs/go/libp2p_node/dht/dhtclient/dhtclient.go

```diff
@@ -306,7 +306,6 @@ func (dhtClient *DHTClient) bootstrapLoopUntilTimeout() error {
 			)
 		case <-ctx.Done():
 			sleepTime = 0
-			break
```

Comment:
pointless

Reply:
see https://github.com/dominikh/go-tools/issues/250

Reply:
it's needed to exit connection loop
but you can set err here. so it will exit by err

Reply:
we exit loop with line 312